src/modules/ekf_att_pos_estimator/ekf_att_pos_estimator_main.cpp

```diff
@@ -1007,7 +1017,7 @@ void AttitudePositionEstimatorEKF::updateSensorFusion(const bool fuseGPS, const
 	}
 
 	// Fuse Airspeed Measurements
-	if (fuseAirSpeed && _ekf->VtasMeas > 7.0f) {
```

Comment:
I would like to understand the rationale for removing the additional check here.